src/admin/settings/settings.html

```diff
@@ -0,0 +1,88 @@
+<h3>{{'admin.platform.settings' | translate}}</h3>
+<form id="settingsFileForm" enctype="multipart/form-data" ng-submit="uploadSettings()">
+    <fieldset>
+        <h4>{{'admin.settings.upload' | translate}}</h4>
+        <div class="form-group">
+            <label for="fileInput">{{'admin.settings.fileChosen' | translate}}</label>
+            <input class="button-file" type="file" id="fileInput" name="file" value="{{'server.button.selectFile' | translate}}" admin-import="settingsFile"/>
+        </div>
+    </fieldset>
+    <input type="submit" value="{{'admin.submit' | translate}}" />
+</form>
+<form id="settingsLocationForm" method="post" ng-submit="uploadFileLocation()">
+    <fieldset>
+        <h4>{{'admin.settings.location' | translate}}</h4>
+        <div class="form-group">
+            <label for="settingsLocation">{{'admin.settings.location.new' | translate}}</label>
+            <input type="text" id="settingsLocation" name="settingsLocation" ng-model="location"/>
+        </div>
+    </fieldset>
+    <input type="submit" value="{{'admin.submit' | translate}}"/>
+</form>
+<form>
+    <fieldset>
+        <h4>{{'admin.settings.importexport.header' | translate}}</h4>
+        <div class="form-group">
+            <label for="exportConfig">{{'admin.settings.importexport.export' | translate}}</label>
+            <input class="button button-export" type="button" id="exportConfig" name="exportConfig" ng-click="exportConfig()" value="{{'admin.settings.importexport.exportButton' | translate}}" />
+        </div>
+    </fieldset>
+</form>
+
+<h3>{{'admin.settings.platform' | translate}}</h3>
+<div ng-repeat="pSettings in platformSettings.settingsList">
+    <form id="platformSettingsForm">
+        <h4>{{'admin.settings.section.'+ pSettings.section | translate}}</h4>
+        <fieldset ng-class="{'form-group': isBool(option)}" ng-repeat="option in pSettings.settings">
```

Comment:
Is there a reason this isn't a `<div class="form-group">` -- a `<fieldset>` with one value doesn't make semantic sense.